app/Http/Requests/Api/Cart/StoreRequest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Requests\Api\Cart;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StoreRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
```

Comment:
Best practice to still check if the user is authenticated even if we already have middlewares on the route.